chef/cookbooks/glance/recipes/api.rb

```diff
@@ -94,6 +94,11 @@
 
 network_settings = GlanceHelper.network_settings(node)
 
+glance_stores = "glance.store.filesystem.Store,glance.store.http.Store,glance.store.cinder.Store"
```

Comment:
I think it looks less ugly if you write it as an array and you simply join it in the template or at template assign.